README.md

```diff
@@ -76,40 +76,44 @@ Put your SSL certificate as `./volumes/web/cert/cert.pem` and the private key th
 no password as `./volumes/web/cert/key-no-password.pem`. If you don't have
 them you may generate a self-signed SSL certificate.
 
-## Starting/Stopping
+### Starting/Stopping
 
-### Start
+#### Start
 ```
 docker-compose start
 ```
 
-### Stop
+#### Stop
 ```
 docker-compose stop
 ```
 
-### Update
+### Removing D
 
-Make sure to backup Mattermost data before proceeding.
+#### Remove the containers
 ```
-docker-compose down
-git pull
-docker-compose build
-docker-compose up -d
+docker-compose stop && docker-compose rm
 ```
 
-## Removing
-
-### Remove the containers
+#### Remove the data and settings of your Mattermost instance
 ```
-docker-compose stop && docker-compose rm
+sudo rm -rf volumes
 ```
 
-### Remove the data and settings of your Mattermost instance
+## Update Docker to Latest Version
+
+First, back up your database and your `./data` file [as described here](https://docs.mattermost.com/administration/backup.html#backup).
```

Comment:
This is not the correct procedure to backup your data when using Docker, you need to backup your mounted volumes. I propose the following, feel free to reword 😉 


First, shutdown your containers to back up your data.
```
docker-compose down
```
Back up your mounted volumes to save your data. If you use the default `docker-compose.yml` file proposed on this repository, your data is on `./volumes/` folder.

Then run the following commands.
```
git pull
docker-compose build
docker-compose up -d
```